api/types/events/events.proto

```diff
@@ -2505,3 +2506,21 @@ message UpgradeWindowStartUpdate {
     UpgradeWindowStartMetadata UpgradeWindowStart = 4
         [ (gogoproto.nullable) = false, (gogoproto.embed) = true, (gogoproto.jsontag) = "" ];
 }
+
+message EC2DiscoveryScriptRun {
```

Comment:
I think this is too specific, how about `SSMRun`? If we want to differentiate further, we could add "type" field or something in future.

Also, missing godoc.